api/bases/keystone.openstack.org_keystoneapis.yaml

```diff
@@ -310,6 +310,14 @@ spec:
                 description: PreserveJobs - do not delete jobs after they finished
                   e.g. to check logs
                 type: boolean
+              processNumber:
+                default: 3
+                description: Processumber - Number of processes running in keystone
+                  API
+                format: int32
+                maximum: 10
```

Comment:
not sure about setting a max here.  @d34dh0r53 @dmendiza @xek   - is there a reasonable setting here or should we just leave it blank?

Reply:
I deleted the maximum limit for now for testing.

Reply:
I also don't think a maximum is needed, it's easy enough to add should this become a problem.